appveyor.yml

```diff
@@ -5,19 +5,94 @@ environment:
   # MinGW
   - name: win32
     platform: mingw
-    qt: 5.9\mingw53_32
+    qt: 5.5\mingw492_32
+    tools: mingw492_32
+  # MinGW
+  - name: win32
+    platform: mingw
+    qt: 5.6\mingw49_32
+    tools: mingw492_32
+  # MinGW
+  #  - name: win32
+  #    platform: mingw
+  #    qt: 5.9\mingw53_32
+  #    tools: mingw530_32
+  # MSVC x32
+  - name: win32
+    platform: x86
+    qt: 5.6\msvc2015
   # MSVC x64
   - name: win64
     platform: amd64
-    qt: 5.9\msvc2015_64
+    qt: 5.6\msvc2015_64
 
 init:
   - if %platform%==mingw set PATH=%PATH:C:\Program Files\Git\usr\bin;=%
-  - if %platform%==mingw set PATH=C:\Qt\Tools\mingw530_32\bin;%PATH%
+  - if %platform%==mingw set PATH=C:\Qt\Tools\%tools%\bin;%PATH%
   - set PATH=C:\Qt\%qt%\bin;%PATH%
   - if not %platform%==mingw call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" %platform%
 
 # Flesh out with amd64/msvc cases later.
 build_script:
-  - qmake GPSBabel.pro
-  - if not %platform%==mingw (nmake) else (mingw32-make)
+  - ps: |
+      if (($env:platform -eq "mingw") -and ($env:qt -eq "5.5\mingw492_32"))
+      {
+        Get-Location
+        qmake -query
+        # mimic creator shadow build to match Inno setup file
+        New-Item ..\build-GPSBabel-Desktop_Qt_5_5_1_Mingw_32bit-Release -type directory -force
+        cd ..\build-GPSBabel-Desktop_Qt_5_5_1_Mingw_32bit-Release
+        qmake ..\gpsbabel\GPSBabel.pro -spec win32-g++
+        mingw32-make qmake_all
+        mingw32-make
+        if ($LastExitCode -ne 0) { $host.SetShouldExit($LastExitCode)  }
+        # copy GPSBabel.exe for use by test_script
+        New-Item ..\gpsbabel\release -type directory -force
+        Copy-Item release\GPSBabel.exe ..\gpsbabel\release\GPSBabel.exe
+        cd ..\gpsbabel
+        New-Item build-app-Desktop_Qt_5_5_1_MinGW_32bit-Release -type directory -force
+        cd build-app-Desktop_Qt_5_5_1_MinGW_32bit-Release
+        qmake ..\gui\app.pro -spec win32-g++
+        mingw32-make qmake_all
+        mingw32-make
+        if ($LastExitCode -ne 0) { $host.SetShouldExit($LastExitCode)  }
+        lupdate ..\gui\app.pro
+        lrelease ..\gui\app.pro
+        # windeployqt in 5.5.1 suffers from bug
+        # https://bugreports.qt.io/browser/QTBUG-48946
+        # which trashes the translations.  Work around
```

Comment:
How gross that was to find! We've been immune from this in release so far becayse I run lupdate and lrelease (and everything else I can...) on my release system (a Mac) which has VMs in it running a shared source tree.